presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -118,6 +119,12 @@ public enum StandardErrorCode
     UNSUPPORTED_ANALYZER_TYPE(0x0001_0029, INTERNAL_ERROR),
     SLICE_TOO_LARGE(0x0001_002A, INTERNAL_ERROR),
     HBO_FAILURE(0x0001_002B, INTERNAL_ERROR),
+    CRYPT_ALGORITHM_ERROR(0x0001_002C, EXTERNAL),
+    CERT_PARSE_ERROR(0x0001_002D, EXTERNAL),
+    PEER_VERIFY_ERROR(0x0001_002E, EXTERNAL),
+    SSL_HANDSHAKE_ERROR(0x0001_002F, EXTERNAL),
+    KEY_MANAGEMENT_ERROR(0x0001_0030, EXTERNAL),
+    KEYSTORE_OPERATION_ERROR(0x0001_0031, EXTERNAL),
```

Comment:
I think we can group these into a single error category related to SSL connection managment.  Please check if one exists that we can use already.